### PR TITLE
Notebook analysis

### PR DESCRIPTION
**Pull request by suganya-s**
Notebook about extracting finding vs thing document types, one analysis A/B test 1, and one investigating an alternative of the bootstrapping method.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,4 +57,11 @@ and a combined file with the overall sample, saved as `full_sample_taxon_ab_2019
 
 Some rounding may result in a very small amount more or less than the k value being included in the final sample, so 
 ideally specify a k a few journeys higher than the k you require.
-                           
\ No newline at end of file
+
+### Finding vs thing pages
```
Comment (mammykins) [inline README.md]: README has been modified in recent PR. This should sit below the sample.py and above analysis.py.

Comment (mammykins) [reply README.md]: I've added this as a requirement, i.e. you need this to do the analysis

**Comment by mammykins** [comment]
does the new bootstrap method cope with 10,000 resamples without crashing the notebook?

**Comment by suganya-s** [comment]
Yep it works on 10,000 :) and takes <2 seconds instead of 9 minutes

**Comment by mammykins** [comment]
haha, nice efficiency gain!

**Pull request closed without merging**